### PR TITLE
Add meta descriptions

### PR DESCRIPTION
**Pull request by lunson**
Adds meta descriptions to the `Header` of the website pages.

This is part of an effort to improve the SEO of the website and increase its website score, which as of this PR, is 80 and needs help specifically in the SEO area: https://website.grader.com/results/openmicroscopy.org

(I did a similar exercise for the GS website and our score has gone up from 79 to 97.)

The descriptions are just a first pass to insure that the website has them present. I did my best to make them meaningful to the content of the page with the caveat that I'm no digital marketing/SEO expert.

Please note: the multiple "clean up white space" commits are the result of my text editor automagically cleaning up white spaces from pages I edit. I've separated these from actual changes to code.

### CODE REVIEW

File: careers/index.html [hunk 1]
```diff
@@ -3,13 +3,13 @@
 title: Careers
 filename: careers
 main-blurb: Join the OME team! We develop unique world-leading applications that are changing the way bioscience works
----          
-                
+meta_description: Develop world-leading applications that are changing the way bioscience works by joining the OME team
```
Comment (mtbc) [inline careers/index.html]:
```suggestion
meta_description: Develop world-leading applications that are changing the way bioscience works by joining the OME team.
```

File: docs/index.html [hunk 2]
```diff
@@ -1,21 +1,22 @@
 ---
 layout: default
 title: Docs
----          
+meta_description: Extensive documentation on the OME model, formats, development, and all of its products
```
Comment (mtbc) [inline docs/index.html]:
```suggestion
meta_description: Extensive documentation on the OME model, formats, development, and all of its products.
```

File: omero/developers/docs.html [hunk 3]
```diff
@@ -2,15 +2,16 @@
 layout: omero-usergroup-devs
 title: OMERO for Developers
 usergroup: developers
---- 
+meta_description: Extensive documentation on the functionality of OMERO
```
Comment (mtbc) [inline omero/developers/docs.html]:
```suggestion
meta_description: Extensive documentation on the functionality of OMERO.
```

File: omero/developers/index.html [hunk 4]
```diff
@@ -2,10 +2,11 @@
 layout: omero-usergroup-devs
 title: OMERO for Developers
 usergroup: developers
----     
+meta_description: Learn more about OMERO for Developers
```
Comment (mtbc) [inline omero/developers/index.html]:
```suggestion
meta_description: Learn more about OMERO for Developers.
```

File: omero/institution/docs.html [hunk 5]
```diff
@@ -2,14 +2,15 @@
 layout: omero-usergroup-inst
 title: Documentation
 usergroup: institutions
---- 
+meta_description: Extensive documentation on the functionality of OMERO
```
Comment (mtbc) [inline omero/institution/docs.html]:
```suggestion
meta_description: Extensive documentation on the functionality of OMERO.
```

File: omero/institution/index.html [hunk 6]
```diff
@@ -2,10 +2,11 @@
 layout: omero-usergroup-inst
 title: OMERO for Your Institution
 usergroup: institutions
----     
+meta_description: Learn more about OMERO for Institutions
```
Comment (mtbc) [inline omero/institution/index.html]:
```suggestion
meta_description: Learn more about OMERO for Institutions.
```

File: omero/scientists/docs.html [hunk 7]
```diff
@@ -2,9 +2,10 @@
 layout: omero-usergroup-sci
 title: OMERO for Scientists
 usergroup: scientists
---- 
-        
-        <hr class="whitespace">        
+meta_description: Extensive documentation on the functionality of OMERO
```
Comment (mtbc) [inline omero/scientists/docs.html]:
```suggestion
meta_description: Extensive documentation on the functionality of OMERO.
```

File: omero/scientists/index.html [hunk 8]
```diff
@@ -2,10 +2,11 @@
 layout: omero-usergroup-sci
 title: OMERO for Scientists
 usergroup: scientists
----     
+meta_description: Learn more about OMERO for Scientists
```
Comment (mtbc) [inline omero/scientists/index.html]:
```suggestion
meta_description: Learn more about OMERO for Scientists.
```

File: on-the-web/index.html [hunk 9]
```diff
@@ -1,7 +1,8 @@
 ---
 layout: default-no-anchor
 title: On the Web
----          
+meta_description: Find OME on social media and catch up on monthly minutes.
```
Comment (mtbc) [inline on-the-web/index.html]:
```suggestion
meta_description: Find OME on social media and catch up on weekly minutes.
```

File: support/index.html [hunk 10]
```diff
@@ -1,16 +1,17 @@
 ---
 layout: default
 title: Support
----          
+meta_description: A collection of support resources to help the OME community
```
Comment (mtbc) [inline support/index.html]:
```suggestion
meta_description: A collection of support resources to help the OME community.
```

**Comment by mtbc** [comment]
:+1: Thank you for separating the white-space commits and for catching those missing attribute "="!

**Comment by lunson** [comment]
Thanks guys!

**Pull request merged**